fe/fe-core/src/main/java/com/starrocks/http/rest/transaction/TransactionWithChannelHandler.java

```diff
@@ -49,7 +49,8 @@ public ResultWrapper handle(BaseRequest request, BaseResponse response) throws S
         Long timeoutMillis = txnOperationParams.getTimeoutMillis();
         String label = txnOperationParams.getLabel();
         Channel channel = txnOperationParams.getChannel();
-        LOG.info("Handle transaction with channel info, label: {}", label);
+        LOG.info("Handle transaction with channel info, label: {}, warehouse: {}", label,
```

Comment:
this log may not be accurate, because only the begin txn will take the warehouse name from parameter, subsequent ops are not using the warehousename parameter at all.